File: Java Programming/Pratice of Java Programming/program2.java
/*
Q2. Accept number from user and display below pattern.
Input : 5
Output : 5 # 4 # 3 # 2 # 1 #
*/

import java.util.Scanner;

class Logic
{
    void Pattern(int iNo)
    {
        int i =0;
        for(i=iNo;i>=1;i--)
        {
            System.out.print(i+"\t#\t");
        }
    }
}

class program2 
{
 public static void main(String[] args) 
 {
    Scanner sobj=new Scanner(System.in);
    int iValue = 0 ;

    System.out.print("Enter number of elements : ");
    iValue=sobj.nextInt();

    Logic obj=new Logic();

    obj.Pattern(iValue);

    sobj.close();
    
 }   
}

File: C++ Programming/Competitive Programming/Coding Challenges 6/program2.cpp
//2.Accept one number from user and print that number of * on screen.
//Output: Enter Number: 5
//        *       *       *       *       *

#include <iostream>
using namespace std;

void Display(int iNo)
{
    //write Updater
    while (iNo > 0)
    {
        cout << "*\t";
        iNo--;
    }
    
}

int main()
{
    int iValue = 0;

    cout << "Enter Number: ";
    cin >> iValue;

    Display(iValue);

    return 0;
}